versions.yaml

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+  nginx:
```

Comment:
It's difficult to know the exact structure, but I would be inclined to have a "higher level" key as this file might grow over time. Similar to https://github.com/kata-containers/runtime/blob/master/versions.yaml, maybe you could do something like:

```
dependencies:
  nginx:
     ...
```

Or if not `dependencies`, maybe `externals` or something. I'd also consider adding in `description` and `format` as we do for the runtime equivalent of this file.

Reply:
agree with @jodh-intel, maybe call them `docker_images`? we later can add all docker images that we use:

```
docker_images:
  description: "Docker images used for testing"

  nginx:
    ....
```